boards/st/nucleo_u575zi_q/nucleo_u575zi_q.dts

```diff
@@ -34,6 +35,44 @@
 	};
 };
 
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/*
+		 * Following flash partition is dedicated to the use of nucleo_u575zi_q
+		 * with TZEN=0 (so w/o TFM).
+		 * The board has in total 2048 KiB of flash memory
+		 */
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 DT_SIZE_K(64)>;
+		};
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x00010000 DT_SIZE_K(928)>;
+		};
+		slot1_partition: partition@f8000 {
+			label = "image-1";
+			reg = <0x000f8000 DT_SIZE_K(928)>;
+		};
+
+		scratch_partition: partition@1e0000 {
```

Comment:
It seems scratch partition is no more used by MCUBoot and could be removed.

Reply:
Updated